__tests__/integration/config.test.ts

```diff
@@ -44,147 +42,91 @@ describe('config', () => {
     expect(await result.text()).toEqual('0\n1\n')
   })
 
-  describe('Logger support', () => {
-    const logLevelKey = 'CLICKHOUSE_LOG_LEVEL'
-    let defaultLogLevel: string | undefined
-    beforeEach(() => {
-      defaultLogLevel = process.env[logLevelKey]
-    })
-    afterEach(() => {
-      if (defaultLogLevel === undefined) {
-        delete process.env[logLevelKey]
-      } else {
-        process.env[logLevelKey] = defaultLogLevel
-      }
-    })
-
-    it('should use the default logger implementation', async () => {
-      process.env[logLevelKey] = 'DEBUG'
-      client = createTestClient()
-      const consoleSpy = jest.spyOn(console, 'debug')
-      await client.ping()
-      // logs[0] are about current log level
-      expect(consoleSpy).toHaveBeenNthCalledWith(
-        1,
-        expect.stringContaining('Got a response from ClickHouse'),
-        expect.objectContaining({
-          request_headers: {
-            'user-agent': expect.any(String),
-          },
-          request_method: 'GET',
-          request_params: '',
-          request_path: '/ping',
-          response_headers: expect.objectContaining({
-            connection: expect.stringMatching(/Keep-Alive/i),
-            'content-type': 'text/html; charset=UTF-8',
-            'transfer-encoding': 'chunked',
-          }),
-          response_status: 200,
-        })
-      )
-      expect(consoleSpy).toHaveBeenCalledTimes(1)
-    })
-
-    it('should provide a custom logger implementation', async () => {
-      process.env[logLevelKey] = 'DEBUG'
-      client = createTestClient({
-        log: {
-          // enable: true,
-          LoggerClass: TestLogger,
-        },
-      })
-      await client.ping()
-      // logs[0] are about current log level
-      expect(logs[1]).toEqual({
-        module: 'HTTP Adapter',
-        message: 'Got a response from ClickHouse',
-        args: expect.objectContaining({
-          request_path: '/ping',
-          request_method: 'GET',
-        }),
-      })
-    })
-
-    it('should provide a custom logger implementation (but logs are disabled)', async () => {
-      process.env[logLevelKey] = 'OFF'
-      client = createTestClient({
-        log: {
-          // enable: false,
-          LoggerClass: TestLogger,
-        },
-      })
-      await client.ping()
-      expect(logs).toHaveLength(0)
-    })
-  })
-
-  describe('max_open_connections', () => {
-    let results: number[] = []
-    afterEach(() => {
-      results = []
-    })
+  // describe('Logger support', () => {
```

Comment:
What do you think about these commented-out tests?

Reply:
Forgot about this one. It should be moved to Node.js-only tests for now.